posts/2017-09-14-serverless-v1.22.0.md

```diff
@@ -0,0 +1,232 @@
+Serverless: Successfully installed "serverless-webpack@latest"
+```
+
+That's it. Serverless will do all the heavy lifting behind the scenes (creating an empty `package.json` file (if not yet present), installing the plugin with its peer dependencies, adding the plugin to the `serverless.yml` file, ...). After doing that we're off the races.
+
+If we want to remove a plugin from our service we simply run the `serverless plugin uninstall` command:
+
+```bash
+root@64d555c6e40a:/app# serverless plugin uninstall --name serverless-webpack
+Serverless: Uninstalling plugin "serverless-webpack" (this might take a few seconds...)
+Serverless: Successfully uninstalled "serverless-webpack"
+```
+
+**Note:** Serverless pulls plugin information from the official [`serverless/plugins` repository](https://github.com/serverless/plugins). Make sure to add yours if you want your plugin to be installable via the CLI as well.
+
+**Aside:** The old way of using `npm` to install / uninstall plugins will continue to work as well!
+
+### SSM Parameters in Serverless Variables
+
+AWS SSM Parameters are a great and recommended way to share encrypted variables between and within services.
+
+Serverless v1.22 introduces support for SSM Parameters via the Serverless Variables system.
+
+Let's assume that we've set the following SSM Parameters:
+
+```bash
+aws ssm put-parameter --name foo --value bar --type SecureString
+aws ssm put-parameter --name bar --value foo --type String
+aws ssm put-parameter --name /foo/bar --value bar --type SecureString
+aws ssm put-parameter --name /bar/foo --value foo --type String
+```
+
+We could access those values in your `serverless.yml` file as follows (where `~true` means that the value will be decrypted):
+
+```yml
+service:
+  name: my-service
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+
+custom:
+  foo: ${ssm:foo}
+  bar: ${ssm:bar~true}
+  fooBar: ${ssm:/foo/bar}
+  barFoo: ${ssm:/bar/foo~true}
+  
+# ...snip...
+```
+
+You can read more about the SSM Parameter support in Serverless Variables in the [corresponding documentation](https://serverless.com/framework/docs/providers/aws/guide/variables#reference-variables-using-the-ssm-parameter-store).
+
+### Update function config when running `deploy function`
+
+The `deploy function` command is an easy and fast way to update the functions code without the need to go through a full `serverless deploy`. `deploy function` will push a function's code directly into the Lamdba function making it way faster than a CloudFormation stack update.
+
+From now on `deploy function` will also update the function's configuration. So making quick changes to your function's settings is even easier.
+
+**Note:** Remember that doing a `deploy function` will put your Stack into an inconsistent state since the function and it's configuration is different than the one which is described via CloudFormation. A subsequent `serverless deploy` will overwrite this state completely.
```

Comment:
ach, I was wrong! Here, it is "its" instead of "it's". I got the possessive/contraction rule mixed up. You were right!

Reply:
Thanks for the update @ryanmurakami 👍

Reply:
Thanks @ryanmurakami 👍

Fixed via https://github.com/serverless/blog/pull/307